Formula/m/mumps.rb

```diff
@@ -0,0 +1,56 @@
+class Mumps < Formula
+  desc "MUltifrontal Massively Parallel sparse direct Solver"
+  homepage "https://mumps-solver.org/"
+  url "https://mumps-solver.org/MUMPS_5.6.2.tar.gz"
+  sha256 "13a2c1aff2bd1aa92fe84b7b35d88f43434019963ca09ef7e8c90821a8f1d59a"
+  license "CECILL-C"
+
+  # Core dependencies
+  depends_on "gcc"
+  depends_on "openblas"
+  fails_with :clang
+
+  def install
+    makefile = "Makefile.G95.SEQ"
```

Comment:
Why not use the one that the ipopt resource used?

Reply:
I can switch if it's a problem for more modern gfortrans, both files worked for me locally.

Reply:
Actually, the non G95 might work a bit better for MacOS (I develop on Linux), let me see if I can remove some of the lines modifying the makefile with that one.